lib/Auth/Process/Mfa.php

```diff
@@ -265,32 +294,37 @@ protected static function hasMfaOptions($mfa)
         return (count($mfa['options']) > 0);
     }
     
-    protected function initComposerAutoloader()
+    /**
+     * See if the user has any MFA options other than the specified type.
+     *
+     * @param string $excludedMfaType
+     * @param array $state
+     * @return bool
+     */
+    public static function hasMfaOptionsOtherThan($excludedMfaType, $state)
     {
-        $path = __DIR__ . '/../../../vendor/autoload.php';
-        if (file_exists($path)) {
-            require_once $path;
+        $mfaOptions = $state['mfaOptions'] ?? [];
+        foreach ($mfaOptions as $mfaOption) {
+            if (strval($mfaOption['type']) !== strval($excludedMfaType)) {
+                return true;
+            }
         }
+        return false;
     }
     
-    protected function initLogger($config)
+    protected function initComposerAutoloader()
```

Comment:
GitHub lined this up poorly: the `initComposerAutoloader()` function wasn't changed.